packages/lesswrong/components/editor/EditorFormComponent.tsx

```diff
@@ -560,7 +560,7 @@ class EditorFormComponent extends Component<EditorFormComponentProps,EditorFormC
     if (!fieldHasCommitMessages) return null;
     
     return <div className={classes.changeDescriptionRow}>
-      <span className={classes.changeDescriptionLabel}>Change description{" "}</span>
+      <span className={classes.changeDescriptionLabel}>Edit summary (Briefly describe your changes):{" "}</span>
```

Comment:
FYI this line feels differently confusing to me, but not worse (partly because "edit" sounds like a verb rather than a noun). Approving PR since it's net positive but I think we could do better on the description label here.

Reply:
I just copied the literal text that Wikipedia uses, since that sure felt like the text that probably had the most iteration done on it.

Reply:
That change seems very good and clear to me.